main.tf

```diff
@@ -36,5 +36,9 @@ module "dcos-forwarding-rule-public-agents" {
     port   = "9090"
   }
 
-  labels = "${var.labels}"
+  labels = "${var.labels}"  
```

Comment:
For the labels, we have previously used this configuration: 
```
  labels = "${merge(var.labels, map("name", format(var.name_format,var.cluster_name),	
                                "cluster", var.cluster_name))}"
```
This will be a consistent template across all the modules within the Universal Installer for GCP. If we can update it here to this it will be good.

Reply:
I can go ahead and make the change you're saying, but I just wanted to clarify that I didn't add `labels` to this repo. And I suspect that someone else might be looking at what to do with `labels` since the Google terraform module upgrade broke it.

Reply:
That's correct. We plan to add this label only in the 0.1.X version of the module and pin the GCP version of 1.X with it. If a user wants to use a module with version 2.X, then a 0.2.X version of the module will have the labels removed.

Reply:
Do you still want me to include the change to labels as part of this PR?

Reply:
It looks like you already had it included in this PR from what I see. If you want to just remove it for now, yes you can do this and keep the google provider.

Reply:
It was just whitespace...I removed the additional whitespace so this PR is just the explicit provider change.